shards/modules/core/wires.cpp

```diff
@@ -244,9 +244,18 @@ struct Wait : public WireBase {
     passthrough = false; // also need this to have proper compose output type
   }
 
+  static SHOptionalString inputHelp() {
```

Comment:
In the case of passthru we should not repeat the string all the time, we should have a string declared in a common place and always reuse that for consistance and less memory usage

Reply:
```
struct CoreInfo {
  static inline SHOptionalString PTInputHelp = SHCCSTR("Passthru input");
  static inline SHOptionalString PTOutputHelp = SHCCSTR("Passthru input")
  ```